### ISSUE TITLE
Background Audio only works on Desktop and Not Mobile

### ISSUE THREAD

**Issue by backlashsid** [seksenov/HWAsmtc#1]
Adding msAudioCategory="BackgroundCapableMedia" to audio/video tags in a Universal Project only works on the PC and not Mobile as expected.

**Comment by seksenov** [comment]
This is correct for Mobile you will need a local/packaged JS file and register it as a background task. I'll work on adding it to the sample.
